ts/features/wallet/store/selectors/index.ts

```diff
@@ -1,35 +1,31 @@
 import * as pot from "@pagopa/ts-commons/lib/pot";
 import { createSelector } from "reselect";
 import { GlobalState } from "../../../../store/reducers/types";
+import { isSomeLoadingOrSomeUpdating } from "../../../../utils/pot";
 import { cgnDetailSelector } from "../../../bonus/cgn/store/reducers/details";
 import { idPayWalletInitiativeListSelector } from "../../../idpay/wallet/store/reducers";
 import { itwLifecycleIsValidSelector } from "../../../itwallet/lifecycle/store/selectors";
 import { paymentsWalletUserMethodsSelector } from "../../../payments/wallet/store/selectors";
-import { WalletCard, walletCardCategories } from "../../types";
-import { isSomeLoadingOrSomeUpdating } from "../../../../utils/pot";
-
-const selectWalletFeature = (state: GlobalState) => state.features.wallet;
```

Comment:
It was syntactic sugar, we have no advantages on using it. I felt that the current file was increasing in size and complexity so I removed extras and non-exported selectors.
But I'm ok on restore it to improve readability. Maybe we can address file complexity in a different way

Reply:
Instead of repeating the same selector, I decided to reduce the scope of the current selectors, just to make them more specific. See here: https://github.com/pagopa/io-app/pull/6570/commits/b42a303c5aa079f02a5f6febecf34bee393a5e10
What do you think?